client/add/add.ctrl.js

```diff
@@ -87,13 +89,16 @@ module.exports = class Add {
       .tap(box => {
         this.addedToast(`Box '${box.name}' created successfully`, 'View', `/box/${box.id}`);
       })
-      .then(res => this.$scope.$apply(this.boxes.push(res)))
+      .tap(box => this.$scope.$apply(this.boxes.push(box)))
+      .then(box => this.$scope.$apply(this.boxesById[box.id].push(box)))
       .catch(this.errorHandler);
   }
 
   pokemon (event) {
     this.watchFullScreen();
     const box = this.selected.selectedBox;
+    const boxCounts = mapValues(this.boxesById, (value) => value.contents.length);
+    const that = this;
```

Comment:
Huh, alright I must have done something else wrong, it appears to work without it now. I'm muchly confused. Ach well, removed.